apps/editors/views.py

```diff
@@ -482,7 +482,12 @@ def queue_fast_track(request):
 def queue_moderated(request):
     rf = (Review.objects.exclude(Q(addon__isnull=True) |
                                  Q(reviewflag__isnull=True))
-                        .filter(editorreview=1)
+                        .filter(editorreview=1,
+                                # Only show reviews for listed add-ons.
+                                # Unlisted add-ons typically won't have
+                                # reviews but they might if their status
+                                # is ever regressed.
+                                addon__status__in=amo.LISTED_STATUSES)
```

Comment:
There's (sadly) some confusion in the code between an add-on being [listed (and not deleted, pending, incomplete...)](https://github.com/mozilla/olympia/blob/48911a1ed3f04b81033206f8b040c573573a859f/apps/constants/base.py#L101) and being [listed (and not unlisted)](https://github.com/mozilla/olympia/blob/48911a1ed3f04b81033206f8b040c573573a859f/apps/addons/models.py#L361)